internal/registry/json.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2021 Tetrate
```

Comment:
where does the name "json" come from?

Reply:
primarily these types represent json from OCI, so I didn't think of a better name, yet